Prace_domowe/Praca_domowa2/Grupa4/Yevhenii_Vinichenko/pd2.ipynb

```diff
@@ -0,0 +1,1442 @@
+{
```

Comment:
<p>Fajnie jakbyś napisał jakie są oznaczenia. Zgaduję, że mean(y_i) to średnia w danej grupie. </p><p> </p><p>Btw. w Markdownie można embeddować Texa przy pomocy symbolu $</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>